spatialdata/__main__.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
this is cool, could the usage of this be documented somewhere in docs?

Reply:
I don't know how to add a documentation for this. Do you have an example in mind I could look at?

Reply:
What does this have to do with the IO hot fix?

Also, is this functionality being added for the first time here? I think this should probably live in a separate package which relies on spatial data, but provides command line utilities.

Reply:
@ivirshup I merged this PR with another one that hasn't merged yet. So the io part is mixed with the rest, sorry.

Reply:
I would keep the CLI momentarily in spatialdata because we have a lot of repos already, but we could move it later on if the CLI gets too big. I plan to just add a few extra functions for now (like exporting the polygons to csv and things like this).

Reply:
Added an issue about documenting the CLI so that we will not forget to do it in the future: https://github.com/scverse/spatialdata/issues/144